### PR TITLE
Updated spec parameters

### PR DESCRIPTION
**Pull request by gts2030**
- Applied [updated spec parameters](https://github.com/superblock-dev/chronos-spec/pull/6/commits/1a0b4da99bdc52693d696381355bdc0502febd36)
- Changed electra in config names to alpaca

### CODE REVIEW

File: encoding/ssz/detect/configfork_test.go [hunk 1]
```diff
@@ -332,7 +332,7 @@ func TestUnmarshalBlock(t *testing.T) {
 	bellav := bytesutil.ToBytes4(params.BeaconConfig().BellatrixForkVersion)
 	capellaV := bytesutil.ToBytes4(params.BeaconConfig().CapellaForkVersion)
 	denebV := bytesutil.ToBytes4(params.BeaconConfig().DenebForkVersion)
-	electraV := bytesutil.ToBytes4(params.BeaconConfig().ElectraForkVersion)
+	electraV := bytesutil.ToBytes4(params.BeaconConfig().AlpacaForkVersion)
```
Comment (syjn99) [inline encoding/ssz/detect/configfork_test.go]:
```suggestion
	alpacaV := bytesutil.ToBytes4(params.BeaconConfig().AlpacaForkVersion)
```

File: encoding/ssz/detect/configfork_test.go [hunk 2]
```diff
@@ -456,7 +456,7 @@ func TestUnmarshalBlindedBlock(t *testing.T) {
...
```
Comment (syjn99) [inline encoding/ssz/detect/configfork_test.go]:
```suggestion
	alpacaV := bytesutil.ToBytes4(params.BeaconConfig().AlpacaForkVersion)
```

File: validator/client/propose_test.go [hunk 3]
```diff
@@ -651,7 +651,7 @@ func testProposeBlock(t *testing.T, graffiti []byte) {
 		},
 		{
 			name:    "electra block",
```
Comment (syjn99) [inline validator/client/propose_test.go]:
```suggestion
			name:    "alpaca block",
```

File: beacon-chain/sync/decode_pubsub_test.go [hunk 4]
```diff
@@ -127,7 +127,7 @@ func TestExtractDataType(t *testing.T) {
 	require.NoError(t, err)
 	denebDigest, err := signing.ComputeForkDigest(params.BeaconConfig().DenebForkVersion, params.BeaconConfig().ZeroHash[:])
 	require.NoError(t, err)
-	electraDigest, err := signing.ComputeForkDigest(params.BeaconConfig().ElectraForkVersion, params.BeaconConfig().ZeroHash[:])
+	electraDigest, err := signing.ComputeForkDigest(params.BeaconConfig().AlpacaForkVersion, params.BeaconConfig().ZeroHash[:])
```
Comment (syjn99) [inline beacon-chain/sync/decode_pubsub_test.go]:
```suggestion
	alpacaDigest, err := signing.ComputeForkDigest(params.BeaconConfig().AlpacaForkVersion, params.BeaconConfig().ZeroHash[:])
```

File: beacon-chain/core/blocks/exit_test.go [hunk 5]
```diff
@@ -286,16 +286,16 @@ func TestVerifyExitAndSignature(t *testing.T) {
...
+					Epoch:           params.BeaconConfig().AlpacaForkEpoch,
 				}
 				signedExit := &ethpb.SignedVoluntaryExit{
 					Exit: &ethpb.VoluntaryExit{
 						Epoch:          params.BeaconConfig().DenebForkEpoch + 1,
 						ValidatorIndex: 0,
 					},
 				}
-				electraSlot, err := slots.EpochStart(params.BeaconConfig().ElectraForkEpoch)
+				electraSlot, err := slots.EpochStart(params.BeaconConfig().AlpacaForkEpoch)
```
Comment (syjn99) [inline beacon-chain/core/blocks/exit_test.go]:
```suggestion
				alpacaSlot, err := slots.EpochStart(params.BeaconConfig().AlpacaForkEpoch)
```

**Review by syjn99 — COMMENTED** [review]
I'm afraid that you missed to change `MaxPendingDepositsPerEpoch` (64 -> 128)

**Comment by gts2030** [comment]
I heard it directly from dan to use 64 yesterday. Spec should be updated.

**Review by syjn99 — APPROVED** [review]

**Pull request merged**